app/controllers/cases/case/subcases/subcase.js

```diff
@@ -7,6 +7,6 @@ export default class CasesCaseSubcasesSubcaseController extends Controller {
 
   @action
   refreshSubcases() {
-    this.router.refresh('cases.case.subcases');
+    this.router.refresh('cases.case.index');
```

Comment:
hmm. Don't we want to keep the subcases route in this case? Otherwise the view might jump from the subcase view to the parliament/publication view after an action has been completed